associated-token-account/program/src/error.rs

```diff
@@ -0,0 +1,26 @@
+//! Error types
+
+use {
+    num_derive::FromPrimitive,
+    solana_program::{decode_error::DecodeError, program_error::ProgramError},
+    thiserror::Error,
+};
+
+/// Errors that may be returned by the program.
+#[derive(Clone, Debug, Eq, Error, FromPrimitive, PartialEq)]
+pub enum AssociatedTokenAccountError {
+    // 0
```

Comment:
I sure hope we don't end up with enough error variants to need this...

Reply:
If it saves even one question of "Associated Token Program has custom error 0x0, what does that mean?", it'll be more than worth it!

Reply:
Hehe, fair enough!